doc/src/manual/documentation.md

```diff
@@ -18,6 +18,11 @@ use indentation and code fences to delimit code examples from text. Technically,
 be associated with any other as metadata; Markdown happens to be the default, but one can construct
 other string macros and pass them to the `@doc` macro just as well.
 
+!!! note
+    Markdown support is enabled by the `Markdown` standard library
```

Comment:
Doesn't "enable" here slightly imply that you should do `using Markdown` or something? Could say "Markdown support is implemented in `Markdown`".